lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1738,6 +1770,27 @@ def test_grade_report(self, persistent_grades_enabled):
                 ignore_other_columns=True,
```

Comment:
Before this line, there's a change that is present in the upstream PR but not in this pull request:
```
            result = CourseGradeReport.generate(None, None, self.course.id, None, 'graded')
            result = CourseGradeReport.generate(None, None, self.course.id, {}, 'graded')
```

It's for the following line:
```
1719    1751            result = CourseGradeReport.generate(None, None, self.course.id, None, 'graded')
```

I can't target it with a comment, so I'm commenting a little below it.